### PR TITLE
add cacheType for fate serving

### PR DESCRIPTION
**Pull request by asdfsx**
Fixes  ISSUE#583

Changes:

1. export  variable cacheType 

2. set model.cache.path to `/root/.fate`， because `/root/.fate` is mounted by pvc

3.

### CODE REVIEW

**Comment by LaynePeng** [comment]
@asdfsx could you sign your commits? Ref: https://github.com/FederatedAI/KubeFATE/blob/master/CONTRIBUTING.md#commit

**Comment by asdfsx** [comment]
DCO passed
please check again

File: helm-charts/FATE-Serving/templates/serving-server-module.yaml [hunk 1]
```diff
@@ -27,8 +27,8 @@ data:
     #serviceRoleName=serving
     # cache
     #remoteModelInferenceResultCacheSwitch=false
-    #cache.type=local
-    #model.cache.path=
+    cache.type={{ .Values.cacheType | default "local" }}
+    model.cache.path=/root/.fate
     # local cache
     #local.cache.maxsize=10000
```
Comment (owlet42) [inline helm-charts/FATE-Serving/templates/serving-server-module.yaml:33]: @asdfsx  `.Values.cacheType` may not pass the correct variable, should be changed to `.Values.servingServer.cacheType`.

**Review by owlet42 — CHANGES_REQUESTED** [review]

File: helm-charts/FATE-Serving/templates/serving-server-module.yaml (hunk 1 above)
Comment (asdfsx) [reply helm-charts/FATE-Serving/templates/serving-server-module.yaml:33]: my mistake
I will update it soon

**Review by owlet42 — APPROVED** [review]

**Pull request merged**